### PR TITLE
docs: blog - three Ways to Integrate Automatic SQL Review into GitHub

### PR DESCRIPTION
**Pull request by tianzhou**

### CODE REVIEW

File: content/blog/integrate-sql-review-into-github.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+tags: Explanation
+featured: true
+description: The three ways to integrate Bytebase SQL Review with GitHub. GitHub Action, GitHub App, and GitOps CI
+---
+
+## Overview
+
+The review of SQL change scripts is a critical part of the database change process, which used to be
+the sole responsibility of the DBA. Developers often submitted the SQL scripts to the DBA just before
+going live, making it difficult to ensure the quality of changes.
```
Comment (milasuperstar) [inline content/blog/integrate-sql-review-into-github.md:15]:
```suggestion
the changes go live, leaving the DBAs minimal time to review the scripts (or request changes), making it difficult to ensure the quality of changes.
```

File: content/blog/integrate-sql-review-into-github.md [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+1. Deploy Bytebase Console and complete the basic configuration
+   This mode requires the deployment of Bytebase Console and import the database enviornment into it.
+   Please refer to [Bytebase Installation Guide](https://www.bytebase.com/docs/get-started/install/overview)
+   for the detailed process.
+
+1. Activate SQL Review CI with one click
+   The GitHub Action can be configured automatically by configuring GitOps Workflow in the Project
+   Configuration page from Bytebase Console.
+
+1. Customizing SQL Review rules
```
Comment (milasuperstar) [inline content/blog/integrate-sql-review-into-github.md:162]:
```suggestion
1. Customizing SQL Review rules

```

File: content/blog/integrate-sql-review-into-github.md [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+
+The Bytebase Console-based integration mode provides more powerful functionality:
+
+1. Unlock more advanced SQL Review rules by connecting with the changed database.
+1. Integrate with the subsequent CD processes.
+
+### Setup
+
+1. Deploy Bytebase Console and complete the basic configuration
+   This mode requires the deployment of Bytebase Console and import the database enviornment into it.
```
Comment (milasuperstar) [inline content/blog/integrate-sql-review-into-github.md:154]:
```suggestion
   This mode requires the deployment of Bytebase Console, and you need to import your database environment into Bytebase.
```

File: content/blog/integrate-sql-review-into-github.md [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+   Please refer to [Bytebase Installation Guide](https://www.bytebase.com/docs/get-started/install/overview)
+   for the detailed process.
+
+1. Activate SQL Review CI with one click
+   The GitHub Action can be configured automatically by configuring GitOps Workflow in the Project
+   Configuration page from Bytebase Console.
+
+1. Customizing SQL Review rules
+   If you are not satisfied with the default SQL Review rules, you can configure it visually in the
```
Comment (milasuperstar) [inline content/blog/integrate-sql-review-into-github.md:163]:
```suggestion
   If you are not satisfied with the default SQL Review rules, you can configure it visually from the
```

File: content/blog/integrate-sql-review-into-github.md [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+❌ **More setup.** Users need to deploy Bytebase Console and prepare the database environment.
+
+❌ **Single presentation.** SQL Review results can only be viewed in the file being reviewed and
+there is no overview of SQL Review results.
+
+## Comparison Table
+
+|                       | GitHub Action                 | GitHub App                                    | GitOps CI                                                          |
+| --------------------- | ----------------------------- | --------------------------------------------- | ------------------------------------------------------------------ |
+| For                   | **Personal and small teams**  | **Personal and small teams**                  | **Teams needing complete database change management**              |
```
Comment (milasuperstar) [inline content/blog/integrate-sql-review-into-github.md:204]: is it better to use "individual" here instead of personal?

**Review by milasuperstar — CHANGES_REQUESTED** [review]

File: content/blog/integrate-sql-review-into-github.md (hunk 5 above)
Comment (tianzhou) [inline content/blog/integrate-sql-review-into-github.md:207]:
```suggestion
| For                   | **Individual and small teams**  | **Individual and small teams**                  | **Teams needing complete database change management**              |
```

Comment (tianzhou) [reply content/blog/integrate-sql-review-into-github.md:207]: True

**Comment by tianzhou** [comment]
PTAL

**Comment by tianzhou** [comment]
Submit now. I will address more comments in separate PR

**Pull request merged**